governance.md

```diff
@@ -247,13 +247,28 @@ information that should not be posted to the public internet.
 ### Subcommittees
 
 The Council can create subcommittees that provide leadership and guidance for
-specific aspects of the project. Like the Council as a whole, subcommittees
-should conduct their business in an open and public manner unless privacy is
-specifically called for. Private subcommittee communications should happen on
-the main private mailing list of the Council unless specifically called for.
+specific aspects of the project. Subcommittees are created using the standard
+voting procedure (e.g. lazy consensus). The members of the subcommittee need
+not all be Steering Council Members, but they must have at least one Steering
```

Comment:
What's the rationale for requiring a steering council member? I think that if the steering council approves the formation of a subcommittee, we can trust that the leaders of the subcommittee will be responsible. We don't necessarily need a steering council member formally on the committee.

Reply:
Yeah this is stolen from conda-forge's governance (though we call them sub-teams). At least the reason in that context is that we wanted the ability to add non-core members to sub-teams. So this relaxes it from effectively requiring all steering council members to at least one on a subcommittee. Though I may be missing the question here.

Reply:
Gotcha. I was reading it from the other direction: why require *any* SC members.

But, this is minor. We can amend this later if this becomes an issue in practice.

Reply:
I see. That makes sense. Some thoughts on why this might be useful...

* A point of contact between the subcommittee and the steering council
* Some overall structure for where subcommittees fit in
* A BDFL delegate that the BDFL has easy access to

The last one is more specific to us. Perhaps there are other reasons. Thoughts?

Reply:
👍 Those are all good reasons.

Reply:
These are good points and give reasoning for the rule.  Should they be added?

Reply:
Personally, I'm comfortable with the rationale being here. If a future reader is interested, the discussion will be available.

Reply:
Thanks all. Am going to go ahead and mark this as resolved. Though please reopen if we missed something.